src/main/java/org/fcrepo/migration/validator/Driver.java

```diff
@@ -82,6 +82,8 @@
             description = "The number of threads for parallel processing. Default 5", defaultValue = "5")
     private int threadCount;
 
+    @CommandLine.Option(names = {"--from-list", "-l"}, order = 10, description = "A list of F3 object ids to validate")
```

Comment:
In the description please make it clear that the `-l` param is a path to a file containing a list of ids.

Reply:
@dbernstein updated the description and name to match the migration-utils, both of which describe the functionality better